content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,53 @@
+---
+title: It is not a choice of the instant messaging app
+date: 2021-01-20 06:30
+author: Edward Maurer
+blog_id: blog
+category: misc
+---
+
+## It is not a choice of the instant messaging app
+
+Recently, several people reached out to me asking what kind of messenger they should be using now. Furthermore, they said that they actually do not understand what they should be concerned about and whether they should switch from one of the commonly known messengers to another. 
+I was wondering how to answer this. Obviously, I could have stood-up and advocated for XMPP (Extensible Messaging and Presence Protocol), but then I thought this alone is not a helpful answer. On the one hand, I see that many people just make a short-term decision about their communication software. On the other hand, I also see they usually do not switch the platform because they made a very evaluated choice for one specific messenger. Many people just switch again and again.
```

Comment:
> they said that they actually do not understand [...]

Howabout "that they don't actually understand [...]"

> On the one hand, [...]

I would have dropped the "the" there.

> they usually do not switch the platform because

I think this would be fine without the "the" here too

Reply:
>> On the one hand, [...]
> I would have dropped the "the" there.

"on the one hand" is correct, as far as I'm aware.

Reply:
As far as "platform" you need the "the" unless it's "platforms".

> they usually do not switch the platform

or 

> they usually do not switch platforms

I do think the second one is better personally.

Reply:
@zash thank you!